src/oneTrust/helpers/oneTrustAssessmentToCsv.ts

```diff
@@ -0,0 +1,33 @@
+import { OneTrustEnrichedAssessment } from '../codecs';
+import { oneTrustAssessmentToCsvRecord } from './oneTrustAssessmentToCsvRecord';
+
+/**
+ * Converts the assessment into a csv entry.
+ *
+ * @param param - information about the assessment and amount of entries
+ * @returns a stringified csv entry ready to be appended to a file
+ */
+export const oneTrustAssessmentToCsv = ({
```

Comment:
nit: would be nice to have some unit tests for these due to complexity but not a big deal

Reply:
Yes! I want to ship some tests before merging ;)